test/cljs/chronograph_web/components/form_test.cljs

```diff
@@ -0,0 +1,120 @@
+(ns chronograph-web.components.form-test
+  (:require [cljs.test :refer-macros [deftest is testing run-tests use-fixtures]]
+            [chronograph-web.components.form :as form]
+            [chronograph-web.test-utils :as tu]
+            [re-frame.core :as rf]
+            [chronograph-web.fixtures :as fixtures]))
+
+(use-fixtures :once fixtures/silence-logging)
+
+(defn- event-with-value
+  [v]
+  (clj->js {:currentTarget {:value v}}))
+
+(def fake-event (clj->js {:preventDefault (constantly nil)}))
+
+(deftest sentence-case-test
+  (testing "it returns hypenated strings in sentence case"
```

Comment:
These testing strings seem to be off. Shouldn't it be "returns sentence cased strings when hyphenated strings are passed in"?